build.gradle

```diff
@@ -10,18 +10,18 @@ repositories {
 }
 
 def dependencyVersions = [
-        dropwizard: '1.3.17'
+        dropwizard: '2.0.0'
 ]
 
 group 'uk.gov.ida'
 version "$dependencyVersions.dropwizard-${System.env.BUILD_NUMBER ?: 'SNAPSHOT'}"
 
 dependencies {
-    compile "io.dropwizard:dropwizard-core:$dependencyVersions.dropwizard",
+    implementation "io.dropwizard:dropwizard-core:$dependencyVersions.dropwizard",
             'net.logstash.logback:logstash-logback-encoder:4.11',
             'ch.qos.logback:logback-access:1.2.3'
```

Comment:
You probably want to include the logstash dependencies in the `api` section - `compile` doesn't expose libraries to consumers and I believe VSP and the Hub consume those through this library.

```
    implementation "io.dropwizard:dropwizard-core:$dependencyVersions.dropwizard"
    
    api "net.logstash.logback:logstash-logback-encoder:4.11",
       "ch.qos.logback:logback-access:1.2.3"
```